src/ducks/subject.js

```diff
@@ -261,9 +261,9 @@ const fetchSubject = (initialFetch = false) => {
         });
     };
 
-    if (checkEmergencySave()) {  //If not, check if there's an emergency save.
+    if (checkEmergencySave(getState().login.user)) {  //If not, check if there's an emergency save.
       dispatch(emergencyLoadWorkInProgress());
-      dispatch(toggleDialog(<DialogOfContinuation />, false, false));
+      dispatch(toggleDialog(<DialogOfContinuation dispatch={dispatch} />, false, false));
```

Comment:
😕

Not sure if adding the `dispatch` like this is "proper", but that's how it works.